src/components/Properties/PropertyDateTime.vue

```diff
@@ -195,6 +201,8 @@ export default {
 		// default load e.g. fr-fr
 		import('moment/locale/' + this.locale)
 			.then(e => {
+				// force locale change to update
+				// the component once done loading
 				this.locale = locale
 			})
 			.catch(e => {
```

Comment:
So does update-input emit for this to happen?

Reply:
I don't get your question :)